do

```diff
@@ -26,11 +26,18 @@ kubeconform() {
 
 # This variable is used, but shellcheck can't tell.
 # shellcheck disable=SC2034
-help_lint_dashboards="Lint the Grafana dashboards"
+help_lint_dashboards="Lint the Grafana dashboards using dashboard-linter."
 lint-dashboards() {
     install-go-bin "github.com/grafana/dashboard-linter@latest"
 
-    ./bin/dashboard-linter lint dashboards/*
+    ./bin/dashboard-linter lint --strict --verbose dashboards/*
+}
+
+# This variable is used, but shellcheck can't tell.
+# shellcheck disable=SC2034
+help_validate_dashboards="Validate the configurations of default Grafana dashboards."
+validate-dashboards() {
+    bash scripts/validate_dashboards.sh
```

Comment:
I thought about leveraging `dashboard-linter`'s custom rules feature for this, but there's a couple drawbacks with it:
- We'd need to copy/reimplement a lot of the CLI features since we can't dynamically add new rules in Go.
- The linter doesn't appear to have access to some fields like the UID outside of the raw spec JSON, so manipulating it in Go would be a bit of a pain compared to `jq`.